### DISCUSSION TITLE
Activating "Snippet View" to render the thumbnail of a PDF file

### DISCUSSION THREAD

**Discussion by LeCyberDucky** [James-Yu/LaTeX-Workshop#3397] [Q&A] [answered]
Hi, 

I'm working on including some figures in my document using `\includegraphics`. When I scroll through the file suggestions, a preview of `.png` files is displayed. For `.pdf` files, however, the message "Please activate Snippet View to render the thumbnail of a PDF file." is displayed. 

How do I actually activate that? I have searched in the extension settings of VS Code, and I have also tried my luck with the auto complete by editing the VS Code settings json file. But I'm unable to find a matching setting by searching for "snippet", "pdf", thumbnail", "preview", and similar keywords. I have also searched for these keywords in https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ and https://github.com/James-Yu/LaTeX-Workshop/wiki/Hover#latex-workshophovercitationenabled, but I can't seem to find anything relevant. 

I feel like I'm missing something obvious, but I simply can't figure this out. 

![image](https://user-images.githubusercontent.com/19520383/181011023-9af4e124-affc-4d7e-8611-1fb4fad2569f.png)

**Comment by yarakos95** [comment, marked as answer]
This problem is resolved by opening the sidebar from TeX in the activity bar and displaying the SNIPPET VIEW. You need to open it manually. Just open the SNIPPET VIEW once and you don't have to keep it visible all the time.

Open the SNIPPET VIEW as shown in the image below.

<img src="https://user-images.githubusercontent.com/75878492/181726608-aa131fbb-d6ad-46c9-b41a-2a072aa297d6.png" width=30%>

**Reply by LeCyberDucky** [reply]
Indeed, that solved. Thanks a bunch! 
I wonder why this behaves like it does.

**Reply by YDX-2147483647** [reply]
Hi @yarakos95,

I've copied your screenshot in [my answer in LaTeX Stack Exchange](https://tex.stackexchange.com/a/742221/266758). Sorry I didn't ask for permission in advance. Please let me know you’d prefer it removed.

Anyway, thanks for your answer! It’s already benefiting others.

**Reply by yarakos95** [reply]
@YDX-2147483647 

Thank you for letting me know. I don't have a problem with this at all. It is important that knowledge is shared and that fewer people have the same problem!

If the link to this discussion is included alongside the image, it will be considered a citation wherever it is reproduced. I think so.

**Comment by Marcool04** [comment]
Shouldn't this be added to the FAQ? Or a bug request be opened to change the wording of the message? Honestly, this seems like a pretty confusing message, which doesn't suggest what needs doing at all in my view. On my machine, there is nothing called "Snippet view", just "Math symbols"... And indeed, opening that fixes the render problem, but thinking to do so when this message comes up is really not a very logical step. I not that 7 people put a 👍️ on the solution here and 3 have on stackexchange, so it has seemingly been confusing to others who went looking for information too...

I'd be happy to provide a PR for a more informative message, possibly coupled with a new FAQ entry on the repo which we could then link to from the message so we can show people with screenshots what they need to do?

Would that be of interest @James-Yu ?
